cli/print/clusters.go

```diff
@@ -10,14 +10,17 @@ import (
 )
 
 // TODO: implement a -o wide, and expose nodecount, vcpus and memory also?
-var clustersTmplSrc = `ID	NAME	K8S DISTRO	K8S VERSION	STATUS	CREATED	EXPIRES
-{{ range . -}}
+var clustersSourceTmpHeader = `ID	NAME	K8S DISTRO	K8S VERSION	STATUS	CREATED	EXPIRES
+`
+
+var clustersTmplSrc = `{{ range . -}}
```

Comment:
I think splitting this will incorrectly align the indents/tabs between columns. Example:
```
ID          NAME            K8S DISTRO    K8S VERSION    STATUS    CREATED                          EXPIRES
1be4df75    josh-test-01    kind          v1.25.3        queued    2023-06-26 19:56:54 +0000 UTC    2023-06-26 20:06:54 +0000 UTC
1be4df75    josh-test-01    kind    v1.25.3    assigned    2023-06-26 19:56:54 +0000 UTC    2023-06-26 20:06:54 +0000 UTC
1be4df75    josh-test-01    kind    v1.25.3    preparing    2023-06-26 19:56:54 +0000 UTC    2023-06-26 20:06:54 +0000 UTC
1be4df75    josh-test-01    kind    v1.25.3    provisioning    2023-06-26 19:56:54 +0000 UTC    2023-06-26 20:06:54 +0000 UTC
1be4df75    josh-test-01    kind    v1.25.3    running    2023-06-26 19:56:54 +0000 UTC    2023-06-26 20:06:54 +0000 UTC
```

Not sure, maybe there is a way to specify a minimum width?